packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -421,126 +423,103 @@ export const makeInnerVault = (
     );
   };
 
-  // Calculate the target level for Collateral for the vaultSeat and
-  // clientSeat implied by the proposal. If the proposal wants Collateral,
-  // transfer that amount from vault to client. If the proposal gives
-  // Collateral, transfer the opposite direction. Otherwise, return the current level.
-  const targetCollateralLevels = seat => {
-    const { vaultSeat } = state;
-    const proposal = seat.getProposal();
-    const startVaultAmount = getCollateralAllocated(vaultSeat);
-    const startClientAmount = getCollateralAllocated(seat);
-    if (proposal.want.Collateral) {
-      return {
-        vault: AmountMath.subtract(startVaultAmount, proposal.want.Collateral),
-        client: AmountMath.add(startClientAmount, proposal.want.Collateral),
-      };
-    } else if (proposal.give.Collateral) {
-      return {
-        vault: AmountMath.add(startVaultAmount, proposal.give.Collateral),
-        client: AmountMath.subtract(
-          startClientAmount,
-          proposal.give.Collateral,
-        ),
-      };
-    } else {
-      return {
-        vault: startVaultAmount,
-        client: startClientAmount,
-      };
-    }
-  };
+  // Was this simple, but you can't subtract `empty` from a missing allocation.
```

Comment:
This is explaining the implementation so let's put it inside the function. Also, it never "was" so,
```suggestion
  // If you could subtract `empty` from a missing allocation it would be simply:
  // toSeat.incrementBy(fromSeat.decrementBy(harden({ [key]: fromLoses })));
  // fromSeat.incrementBy(toSeat.decrementBy(harden({ [key]: fromGains })));
```
Though I think the whole code example is superfluous. Just remove this block and comment why the `isEmpty()` check is needed. I'll make that suggestion below.